code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -311,6 +317,10 @@
 	if((isnull(user) || istype(user)) && state_open && !panel_open)
 		flick("pod-close-anim", src)
 		..(user)
+		var/mob/living/L = occupant
+		if(L && istype(L))
+			ADD_TRAIT(L, TRAIT_FORCE_STAND, CRYO_TRAIT)
+			L.update_mobility()
```

Comment:
Ah one other thing, you need to remove this trait on destroy

Reply:
Now that I think about it, it should probably be done on `Exited` so that teleporting won't break things as well. If done on `Exited`, is there a need to do it on destroy too?

Reply:
Tested locally, `Exited` works